src/utils.js

```diff
@@ -399,70 +400,31 @@ export function parseTodaysHamkaMenu(rawMenu) {
   return menu;
 }
 
-function getIndicesOf(menuStrFrom, menuStrTo, str) {
-  const menuStrFromLen = menuStrFrom.length;
-  const menuStrToLen = menuStrTo.length;
-  if (menuStrFromLen === 0 || menuStrToLen === 0) {
-    return [];
-  }
-  let startIndex = 0;
-  let index;
-  const indices = [];
-  while ((index = str.indexOf(menuStrFrom, startIndex)) > -1) {
-    const from = index;
-    const to = str.indexOf(menuStrTo, from + menuStrFromLen);
-    startIndex = to + menuStrToLen;
-
-    indices.push({from, to});
-  }
-  return indices;
+function normalizeWhitespace(str) {
+  return str.split('\n').map((l) => l.trim()).filter((l) => l.length > 0);
```

Comment:
This returns array. Shouldn't it be string?

Reply:
Yes, it definitely should be.